### PR TITLE
refactor: remove the expiry field

### LINKED ISSUE #679: Stop exposing expiry timestamp
## Description
As a developer,
I need to deprecate `expiry` timestamp,
so that I can remove misleading information fro API definition.

I confirmed that the `expiry` field is not being used

## Mocks

image

### Update [Required]
- [ ] Remoove references to `expiry` timestamp field from POST /login

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by divyanshugit**
### Description

Removed the `expiry` field from login response model

Fixes #679

### Type of Change:

- Code

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] Removed references to expiry timestamp field from POST /login

**Code/Quality Assurance Only**

- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/api/models/user.py [hunk 1]
```diff
@@ -151,37 +151,6 @@ def add_models_to_namespace(api_namespace):
...
 )
-
-# TODO: Remove 'expiry' after the android app refactoring.
-login_response_body_model = Model(
-    "Login response data model",
-    {
-        "access_token": fields.String(required=True, description="User's access token"),
-        "expiry": fields.Float(
-            required=True, description="Access token expiry UNIX timestamp"
-        ),
```
Comment (isabelcosta) [inline app/api/models/user.py]: these are the only lines that should be removed in this pull request :) All the rest you removed, you should keep

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@divyanshugit You are almost there :) You just have to remove 3 lines of code, after that it the work should be complete.

File: app/api/models/user.py (hunk 1 above)
Comment (divyanshugit) [reply app/api/models/user.py]: Thanks for your review :)

**Comment by isabelcosta** [comment]
@divyanshugit can you please only commit what you have done in this PR? From what I understand your code should only include 3 lines of code. A recommendation we have in our "Contributions Guidelines" is to create a branch specific to an issue you are solving. I think you are accumulating commits you did on a previous PR. The problem is that we squash commits, so the commit that goes to `develop` branch is different than the ones you have in your `develop` branch from previous PR.

**Comment by divyanshugit** [comment]
@isabelcosta  Sorry, I didn't saw that It was just updated before my PR. I just copied the code from mentorship-backend/app/api/resources/user.py / . Again, Sorry for my mistake I will make sure that it will not happen again in future.

**Comment by divyanshugit** [comment]
@isabelcosta  Is this okay ? Can you please suggest me some new issues on which I can start working !

**Review by vj-codes — COMMENTED** [review]
@divyanshugit A good implementation would be to checkout to a suitable branch and then send a PR when you will start working on other issues!

**Review by isabelcosta — DISMISSED** [review]
Great work @divyanshugit ! Thank you for your contribution! Now I just need one little test to see if login is working as expected :)

**Review by vj-codes — DISMISSED** [review]

**Comment by isabelcosta** [comment]
@anitab-org/qa-team to test this, you can see if Login API works well as before, that should be enough :)

**Review by SanketDG — CHANGES_REQUESTED** [review]
There is a `TODO` on https://github.com/anitab-org/mentorship-backend/pull/704/files#diff-a831ce61f9c82aab25c2f42e88908329R155, we may also want to remove that.

Good to merge after that!

**Comment by isabelcosta** [comment]
@SanketDG I don't see such TODO, the current version of the code that @divyanshugit changed  does not have TODO

**Comment by SanketDG** [comment]
@isabelcosta I still see a TODO on line 155 of the changed file

**Comment by isabelcosta** [comment]
@SanketDG very nice catch! At first viewing I did not see it, I had to click to expand, your change request makes sense!

image

@divyanshugit can you please remove the line 155?

**Review by SanketDG — APPROVED** [review]
Looks good :rocket:

**Review by isabelcosta — DISMISSED** [review]
Great work @divyanshugit 🎉

**Review by SanketDG — DISMISSED** [review]
Nicely done! @divyanshugit

**Comment by isabelcosta** [comment]
@anitab-org/qa-team can you please test this PR?

**Review by isabelcosta — APPROVED** [review]

**Review by SanketDG — APPROVED** [review]
Ready for Merge :tada: 

@divyanshugit Tip: You do not need to revert your own commit in your own PR! :P

**Comment by divyanshugit** [comment]
@SanketDG I mistakenly commit in the wrong branch. I was solving the issue #632 in that commit. Sorry for that mistake.

**Review by foongminwong — APPROVED** [review]
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses were tested as below:

   * _Before PR 704_  
    _Screenshot/gif/url_:  

    
       before-ms-backend-pr-704

    _Expected Result_:  As a user, I can view the `expiry` timestamp on MS Swagger UI.
    _Actual Result_: Same as expected.

   * _After PR 704_  
    _Screenshot/gif/url_:  

  
     after-ms-backend-pr-704


    _Expected Result_:  The `expiry` timestamp on MS Swagger UI is removed. As a user, I shouldn't see it on MS Swagger UI.
    _Actual Result_: Same as expected.

3. Comments: Good work @divyanshugit !!
 
4. OS Version: Windows 10

**Comment by isabelcosta** [comment]
@foongminwong your test report is so amazing 👏

**Pull request merged**